xtask/src/commands/release/plan.rs

```diff
@@ -120,6 +120,23 @@ pub fn plan(workspace: &Path, args: PlanArgs) -> Result<()> {
             } else {
                 ReleaseType::Minor
             };
+
+            // Special case: some packages are perma-unstable, meaning they won't ever have a stable
+            // release. For these packages, we always use a patch release.
+            let amount = match package {
+                Package::EspRomSys => {
+                    if amount != ReleaseType::Patch {
```

Comment:
```suggestion
                Package::EspRomSys if amount != ReleaseType::Patch => {
```

You can collapse things a bit more.